torch/fx/experimental/migrate_gradual_types/constraint_generator.py

```diff
@@ -580,10 +617,9 @@ def generate_constraints_node(self, n: Node, counter):
             else:
                 raise RuntimeError(f'No inference rule registered for target {n.target}!')
 
-        # TODO
+        # TODO: verify that no constraint should be generated here
```

Comment:
Turn this into an assert?

Reply:
I'm still thinking about this point. Should there be a constraint for get_attr? so I will figure it out and make a PR for it.